specs/Freeze.md

```diff
@@ -0,0 +1,176 @@
+# Background
+Edge browser has a sleeping tab feature which reduces resource usage when the tab is in the background. We are introducing WebView2 APIs
+to access this feature so that invisible WebView can use less resources. We'd appreciate your feedback.
+
+
+# Description
+You may call the `TrySuspendAsync` API to have the WebView2 consume less memory. This is useful when your Win32 app becomes invisible, or when your Universal Windows Platform app is being suspended, during the suspended event handler before completing the suspended event.
+
+# Examples
+## .Net, WinRT
+```c#
+async protected void OnSuspending(object sender, SuspendingEventArgs args)
+{
+    SuspendingDeferral deferral = args.SuspendingOperation.GetDeferral();
+    // Ensure that CoreWebView2Controller is invisible, it must be invisible for TrySuspendAsync to succeed.
+    // webView here is Microsoft.Web.WebView2.Wpf.WebView2.
+    webView.Visibility = false;
```

Comment:
Visibility isn't a boolean, this needs to be set to `Visibility.Collapsed`.

Reply:
Updated the sample to use WinUI webview2 control and moved code for WPF and Winforms WebView2 control into comment.